pages/callbacks/chaoss_callbacks.py

```diff
@@ -100,7 +100,7 @@ def create_graph(data, contribs, view):
         return None
```

Comment:
files should be taken off of PR as nothing has been changed

Reply:
( as before ) it's going to end up in some PR somewhere because it's based on the Black python formatter.

Reply:
it's going to end up in some PR somewhere because it's based on the Black python formatter.